src/runtime-rs/crates/resource/src/cdi_devices/mod.rs

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (c) 2024 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+pub mod container_device;
+
+use agent::types::Device;
+use std::collections::HashMap;
+
+#[derive(Clone, Default)]
+pub struct DeviceInfo {
+    pub class_id: String,
+    pub vendor_id: String,
+    pub host_path: String,
+    pub device_options: Vec<String>,
+}
+
+#[derive(Clone, Default)]
+pub struct ContainerDevice {
+    pub device_info: Option<DeviceInfo>,
+    pub device: Device,
+}
+
+lazy_static! {
+    // Prefix of vendorID-classID -> vendor/class
+    pub static ref CDI_DEVICE_VENDOR_CLASS: HashMap<&'static str, &'static str> = {
+        let mut m = HashMap::new();
+        m.insert("0x10de-0x030", "nvidia.com/gpu");
+        m.insert("0x8086-0x030", "intel.com/gpu");
+        m.insert("0x1002-0x030", "amd.com/gpu");
+        m.insert("0x15b3-0x020", "mellanox.com/nic");
```

Comment:
```suggestion
        m.insert("0x15b3-0x020", "nvidia.com/nic");
```
refer: https://admin.pci-ids.ucw.cz/read/PC/15b3

>Name: Mellanox Technologies
Note: now NVIDIA
AlbertP
2022-07-01 20:48:20